internal/xds/rbac/rbac_engine.go

```diff
@@ -110,14 +114,17 @@ func (cre *ChainEngine) IsAuthorized(ctx context.Context) error {
 
 // engine is used for matching incoming RPCs to policies.
 type engine struct {
-	policies map[string]*policyMatcher
+	policyName string
+	policies   map[string]*policyMatcher
 	// action must be ALLOW or DENY.
-	action v3rbacpb.RBAC_Action
+	action         v3rbacpb.RBAC_Action
+	auditLoggers   []audit.Logger
+	auditCondition v3rbacpb.RBAC_AuditLoggingOptions_AuditCondition
 }
 
 // newEngine creates an RBAC Engine based on the contents of policy. Returns a
```

Comment:
Nit: "based on the contents of _a policy_" (or "based on the contents of _config_"?)